demo/src/screens/componentScreens/SharedTransitionScreen.js

```diff
@@ -0,0 +1,178 @@
+import React, {Component} from 'react';
+import {LayoutAnimation, FlatList} from 'react-native';
+import {Colors, View, Image, TouchableOpacity, SharedTransition, Card, Button, Text, Modal} from 'react-native-ui-lib'; //eslint-disable-line
+
+const DATA = [
+  {
+    id: '12',
+    title: 'white pendant lamp',
+    location: 'Paris, France',
+    views: 1224,
+    image:
+      'https://images.unsplash.com/photo-1513506003901-1e6a229e2d15?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1500&q=80',
+  },
+  {
+    id: '13',
+    title: 'black floor lamp at the corner',
+    location: 'Paris, France',
+    views: 302,
+    image:
+      'https://images.unsplash.com/photo-1494438639946-1ebd1d20bf85?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1494&q=80',
+  },
+  {
+    id: '14',
+    title: 'gray desk lamp near white wall',
+    location: 'Paris, France',
+    views: 554,
+    image:
+      'https://images.unsplash.com/photo-1507473885765-e6ed057f782c?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=668&q=80',
+  },
+  {
+    id: '15',
+    title: 'round white and yellow bowl illustration',
+    location: 'Paris, France',
+    views: 1999,
+    image:
+      'https://images.unsplash.com/photo-1526308430620-59f514ed2152?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1500&q=80',
+  },
+  {
+    id: '16',
+    title: 'black and orange polka dot print textile',
+    location: 'Paris, France',
+    views: 2014,
+    image:
+      'https://images.unsplash.com/photo-1544070643-24128d1f6033?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=668&q=80',
+  },
+];
+
+const ANIMATION_DURATION = 200;
+
+export default class PlaygroundScreen extends Component {
+  constructor(props) {
+    super(props);
+
+    this.cards = [];
+  }
+
+  state = {
+    showDetails: false,
+  };
+
+  componentDidUpdate() {
+    LayoutAnimation.configureNext({
+      ...LayoutAnimation.Presets.easeInEaseOut,
+      duration: ANIMATION_DURATION,
+    });
+  }
+
+  focusItem = id => {
```

Comment:
Where do you use 'focusItem()'?

Reply:
you're right, I forgot to clean the example screen. most of the code moved to the component.